controller/network/routesender.go

```diff
@@ -148,6 +126,61 @@ attendance:
 	return peerData, nil, nil
 }
 
+func (self *routeSender) handleRouteSend(attempt uint32, path *Path, strategy xt.Strategy, status *RouteStatus, terminator xt.Terminator, logger *pfxlog.Builder) (peerData xt.PeerData, cleanups map[string]struct{}, err error) {
+	if status.ErrorCode != nil {
```

Comment:
statusError != nil should only happend if success = false, so the code should reflect that in some way